Feature: Add option to disable interpolation in ARKODE (#474)
LLNL/sundials: doc/shared/RecentChanges.rst

compilation issues with the Cray Fortran compiler.

Fixed a bug in the HIP execution policies where ``WARP_SIZE`` would not be set
with ROCm 6.0.0 or newer.

Fixed a bug that caused error messages to be cut off in some cases. Fixes
`GitHub Issue #461 <https://github.com/LLNL/sundials/issues/461>`_.

Fixed a memory leak when an error handler was added to a
:c:type:`SUNContext`. Fixes `GitHub Issue #466
<https://github.com/LLNL/sundials/issues/466>`_.

Fixed a bug where :c:func:`MRIStepEvolve` would not handle a recoverable error
produced from evolving the inner stepper.

Added missing ``SetRootDirection`` and ``SetNoInactiveRootWarn`` functions to
ARKODE's SPRKStep time-stepping module.

Fixed a bug in :c:func:`ARKodeSPRKTable_Create` where the coefficient arrays
were not allocated.

Fix bug on LLP64 platforms (like Windows 64-bit) where ``KLU_INDEXTYPE`` could be
32 bits wide even if ``SUNDIALS_INT64_T`` is defined.

Check if size of ``SuiteSparse_long`` is 8 if the size of ``sunindextype`` is 8
when using KLU.

**Deprecation Notices**

Numerous ARKODE stepper-specific functions are now deprecated in favor of
ARKODE-wide functions.

> Deprecated ``ARKStepSetOptimalParams`` function. This functions does not have an

Reviewer: ```suggestion
Deprecated the `ARKStepSetOptimalParams` function. Since this function does not have an
```
